frontend/packages/core/src/card.tsx

```diff
@@ -39,26 +39,33 @@ const StyledCardHeaderContainer = styled.div({
 });
 
 const StyledCardHeader = styled(Grid)({
-  padding: "16px",
-  minHeight: "72px",
+  padding: "6px 0px",
+  minHeight: "48px",
   ".MuiGrid-item": {
     padding: "0px 8px",
   },
 });
 
-// TODO:: confirm the width and height of the icon container
+const StyledHeaderAvatarContainer = styled.div({
+  padding: "12px",
+  height: "48px",
+  width: "48px",
+  alignSelf: "center",
+});
+
 // TODO: use material ui avatar component and implement figma design
-const StyledCardHeaderAvatar = styled.div({
-  width: "32px",
-  height: "32px",
+const StyledHeaderAvatar = styled.div({
+  width: "24px",
+  height: "24px",
   marginRight: "16px",
+  fontSize: "18px",
```

Comment:
fyi didn't use the typography component b/c we don't have a matching variant. (font sizes for icons in the core design system are 18px, 24px, and 36x)